builtin/logical/pki/cert_util.go

```diff
@@ -747,6 +747,8 @@ func signCert(b *backend,
 			return nil, errutil.UserError{Err: "RSA keys < 2048 bits are unsafe and not supported"}
 		}
 
+	default:
+		return nil, errutil.InternalError{Err: fmt.Sprintf("supported key type value: %s", data.role.KeyType)}
```

Comment:
Should that be "unsupported key type..."?

Reply:
Oops good eye, will fix.